### PR TITLE
 Home page of my mobile version portfolio.

### PR DESCRIPTION
**Pull request by Taher-web-dev**
I've created a html file and css file for the homepage of my mobile version portfolio. I have ensured that my code respect the linter requirements . Please take  a look and give me your feedback before to merge with master branch.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        <link rel="stylesheet" type="text/css" href="styles/style.css">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+    </head>
+    <body>
+        <nav>
+            <a class="logo" href="#">My Logo</a>
+            
+            <img src="statics/images/Union.png" alt="Menu">
+               
+        </nav>
```
Comment (karvel-code) [inline index.html:15]: - [ ] This section is supposed to be wrapped inside a `header` tag in order to enhance HTML semantics. Please ensure you do so. 
- [ ] Please ensure that you follow the Figma design in assigning the margins as well as positioning the different elements within the page.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        <meta charset="utf-8">
+        <title> Taher Haggui </title>
+        <link rel="stylesheet" type="text/css" href="styles/style.css">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+    </head>
+    <body>
+        <nav>
+            <a class="logo" href="#">My Logo</a>
+            
+            <img src="statics/images/Union.png" alt="Menu">
```
Comment (karvel-code) [inline index.html:13]: - [ ] The hamburger icon on the right should be wrapped inside the `<nav>`tag for semantics reasons.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+                If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <footer>
+                <h3>LET'S CONNECT</h3>
+                <div>
+                   <a class="icon" href="#"><img src="statics/images/twitter.png" alt="twitter link"></a>
+                   <a class="icon" href="#"><img src="statics/images/Linkedin_icon.png" alt="linked link"></a>
+                   <a class="icon" href="#"><img src="statics/images/M.png" alt="M"></a>
+                   <a class="icon" href="#"><img src="statics/images/github.png" alt="github link"></a>
+                   <a class="icon" href="#"><img src="statics/images/doits.png" alt="doit"></a>
```
Comment (karvel-code) [inline index.html:31]: - [ ] Well done on wrapping the icons inside the `<a>` tags. However, also wrap them inside the `<ul>` and `<li>` tags.

**Review by karvel-code — CHANGES_REQUESTED** [review]
Hi @Taher-web-dev , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All your linter checks pass successsfully.'
- You have included a very professional README file.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 2 above)
Comment (Taher-web-dev) [reply index.html:13]: Do you mean that I should create new nav  tag that can wrap the humberguer icon and which in his turn wrapped inside the first tag.

File: index.html (hunk 3 above)
Comment (Taher-web-dev) [reply index.html:31]: Could you tell me the utilities to use such wrapping structure in this case.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+                I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <footer>
+                <h3>LET'S CONNECT</h3>
+                <div>
+                   <ul>
+                     <li class="icon"><a href="#"><img src="statics/images/twitter.png" alt="twitter link"></a></li>
+                     <li class="icon"><a href="#"><img src="statics/images/Linkedin_icon.png" alt="linked link"></a></li>
```
Comment (safafa) [inline index.html:31]: Original Design : 
image
Your webpage : 
image

- [ ] As you can see from the pictures, your project looks a bit different from the original design. The icons should be aligned to the left and need more space between them. The logo and the menu icon need more space between them. The Headline section needs more margin-left. Kindly make this change respecting the original design is one of the project requirements as it is an important skill for a front-end developer

**Review by safafa — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED ♻️ 

Hi @Taher-web-dev, Good job working on some of the changes requested by the previous reviewer 👏🏼 👏🏼 

But,  there are still some which aren't addressed yet.

## Required changes :

- [ ] Check the inline comments.

## Optional changes :
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Adding a screenshot on your project in the Readme file to make it more descriptive.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by DanSam5K — APPROVED** [review]
# Hi @Taher-web-dev , Approved ✌️ 🥇 
I really admired your persistence, and never give up attitude 👏   I can see you really invested much time and effort trying to perfect and fix your project as suggested by the previous reviewers, keep it up! 👍 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉  

### [Optional] Suggestions 📗 🌱 🌱  
[optional] is not mandatory, but it would be great to try and ac on them as it will help us becomes top professionals
remember we as professionals are always open to improvement, suggestions and growth 🤝 
- [ ] It would be nice to add descriptive documentation to your work, all links are not broken, a screenshot of your project and also a live demo version of the project. 😉  

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**